app/src/main/java/com/example/cinemapp/util/mappers/ActorDetailsMapper.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.cinemapp.util.mappers
+
+import com.example.cinemapp.data.CastMovieCreditDTO
+import com.example.cinemapp.data.PersonDetailsDTO
+import com.example.cinemapp.ui.main.model.CastMovieCredit
+import com.example.cinemapp.ui.main.model.PersonDetails
+import java.time.LocalDate
+
+class ActorDetailsMapper(
+    private val urlMapper: URLMapper
+) {
+    fun mapDateStringListToDescendingYearsList(dates: List<String?>?): List<Int?> {
```

Comment:
Since reading `dates: List<String?>?` already tells you itself that it is a list of strings as a date, you can name this method `mapToDescendingYears`